STYLE-GUIDE.md

```diff
@@ -1,2 +1,77 @@
 ### Refer to Airbnb JavaScript Style Guide
-  - https://github.com/airbnb/javascript
\ No newline at end of file
+  - https://github.com/airbnb/javascript
+
+
+### Writing the React Component
+Prefer to use class-based components instead of pure function,
```

Comment:
Rewrite the wording as following:
"Class-based components are preferable over the ones in pure function..."